app/main/posts/detail/post-video-value.directive.js

```diff
@@ -26,6 +26,8 @@ function PostVideoValueController(
     activate();
 
     function activate() {
-        $scope.videoUrl = $sce.trustAsResourceUrl($scope.videoUrl[0]);
+        if ($scope.videoUrl && $scope.videoUrl.length > 0) {
```

Comment:
if this is not the case, we just then let the videoUrl continue as undefined / empty so it doesn't render